src/main/java/ch/njol/skript/sections/SecLoop.java

```diff
@@ -74,6 +74,11 @@ public boolean init(Expression<?>[] exprs,
 			expr = new ContainerExpression((Expression<? extends Container<?>>) expr, type.value());
 		}
 
+		if (expr.isSingle()) {
+			Skript.error("Can't loop " + expr + " because it's only a single value.");
```

Comment:
```suggestion
			Skript.error("Can't loop " + expr + " because it's only a single value");
```
We don't do full stops in Skript's errors